client/src/app/site/pages/meetings/modules/list-of-speakers-content/components/list-of-speakers-content/list-of-speakers-content.component.html

```diff
@@ -391,6 +391,47 @@
     </div>
 </mat-card>
 
+<!-- Moderator Note-->
+<mat-card class="spacer-bottom-60" [ngClass]="isEditing ? 'os-form-card' : 'os-card'" *ngIf="canSeeModerationNote">
```

Comment:
I don't think it makes sense to display an empty moderation note if the user cannot edit it. 

```suggestion
<mat-card class="spacer-bottom-60" [ngClass]="isEditing ? 'os-form-card' : 'os-card'" *ngIf="canSeeModerationNote && (canManageModerationNote || (moderatorNotes | async))">
```